src/TabBar.js

```diff
@@ -25,7 +25,11 @@ class TabBar extends Component {
         `No action is defined for name=${el.props.name} ` +
         `actions: ${JSON.stringify(Object.keys(Actions))}`);
     }
-    Actions[el.props.name]();
+    if (el.props.onPress){
```

Comment:
People coming other frameworks where they could just put there the action on the class itself mentioned as a string, onPress='onMyButtonPress' where onMyButtonPress is a method defined within the class :) well i'm sure it will rarely happen, it is more as a sanity check for avoiding type missmatch exception. I do not have anything in mind merging this in like it is though.